src/main/java/org/jboss/tools/intellij/openshift/telemetry/TelemetryService.java

```diff
@@ -61,4 +65,12 @@ public TelemetryMessageBuilder getBuilder(){
         return instance.builder.get();
     }
 
+    public static void asyncSend(ActionMessage message) {
```

Comment:
is a usage of upperclass instance possible here ?  doesn't seems to have something specific to either ActionMessage nor FeedbackMessage.

Reply:
not possible, unfortunately. I have made the super class/interface private :(